Bottom: c096dcff0d5d97207fb65f8ca607b8a29148e67e
Top:    9dd8ba614fbbe9d82b591d8736ae29f0d34396a7
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-07-18 14:11:42 -0700

Refresh of dbg

---

diff --git a/examples/rstream.c b/examples/rstream.c
index 8595657..a699e0a 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -181,6 +181,7 @@ static void show_perf(void)
 
 static int size_to_count(int size)
 {
+	return 100;
 	if (size >= 1000000)
 		return 100;
 	else if (size >= 100000)
diff --git a/src/rsocket.c b/src/rsocket.c
index 68361f5..1d293f7 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -361,7 +361,9 @@ static int rs_init_bufs(struct rsocket *rs)
 	if (!rs->sbuf)
 		return -1;
 
+	printf("rs init bufs cm_id %p pd %p\n", rs->cm_id, rs->cm_id->pd);
 	rs->smr = rdma_reg_msgs(rs->cm_id, rs->sbuf, rs->sbuf_size);
+	printf("rs init bufs - after reg %p\n", rs->smr);
 	if (!rs->smr)
 		return -1;
